core/templates/dev/head/pages/exploration_player/audio_controls_directive.html

```diff
@@ -1,37 +1,37 @@
 <div class="audio-controls">
-  <i class="fa fa-cog audio-control-button-icon"
+  <i class="fa fa-cog audio-controls-button-icon"
      ng-if="extraAudioControlsAreShown"
      ng-click="openAudioTranslationSettings()"></i>
-  <img class="audio-control-button-image"
+  <img class="audio-controls-button-image"
        style="margin-top: 2px"
        ng-if="extraAudioControlsAreShown"
        ng-src="<[IMAGE_URL_REWIND_AUDIO_BUTTON]>"
        ng-click="rewindAudioFiveSec()">
-  <i class="fa audio-control-button-icon"
+  <i class="fa audio-controls-button-icon"
      style="min-width: 24px; text-align: left;"
      ng-click="playPauseAudioTranslation()"
-     ng-class="{'fa-volume-off': !AudioPlayerService.isPlaying(), 'fa-volume-up': AudioPlayerService.isPlaying(), 'audio-not-available': !isAudioAvailableInCurrentLanguage()}"
-     tooltip="<[!isAudioAvailableInCurrentLanguage() ? audioNotAvailableMessage : '']>"
+     ng-class="{'fa-volume-off': !AudioPlayerService.isPlaying(), 'fa-volume-up': AudioPlayerService.isPlaying(), 'audio-controls-audio-not-available': !isAudioAvailableInCurrentLanguage()}"
+     tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate) + ' ' + currentAudioLanguageDescription : '']>"
```

Comment:
Any way you can make "currentAudioLanguageDescription" a _parameter_ of the I18N thing, similar to how I18N_FORMS_TYPE_NUMBER_AT_LEAST handles minValue? In other languages, currentAudioLanguageDescription may not always appear at the end.

If not, let's just not i18n it for now. I'm not sure how to do this sort of thing in tooltips, it might be the first time we've had to handle it.